docs/user/pings/metrics.md

```diff
@@ -0,0 +1,182 @@
+# The `metrics` ping
```

Comment:
This is not implemented. Please link to https://bugzilla.mozilla.org/show_bug.cgi?id=1551159